### PR TITLE
BiT Image Classification Quantization NNCF notebook

### PR DESCRIPTION
**Pull request by spradeep28**
This notebook tutorial demonstrates the Quantization of the BiT Image Classification model, which is fine-tuned on the Imagentte dataset with 10 class labels with NNCF. It uses BiT-M-R50x1/1 model, which is trained on ImageNet-21k. Big Transfer (BiT) is a recipe for pre-training image classification models on large supervised datasets and efficiently fine-tuning them on any given target task. The recipe achieves excellent performance on a wide variety of tasks, even when using very few labeled examples from the target dataset. This tutorial uses OpenVINO backend for performing model quantization in NNCF.

### CODE REVIEW

**Comment by spradeep28** [comment]
<p>NNCF Fix <a href="https://github.com/openvinotoolkit/openvino/pull/22236" target="_blank">https://github.com/openvinotoolkit/openvino/pull/22236</a> got merged last week and is part of 2024.0. So had to use openvino-nightly.</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1571/discussion/#comment-1912054769'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by spradeep28** [comment]
<p>Done - Generated table of contents with table_of_content.py</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1571/discussion/#comment-1912054754'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by spradeep28** [comment]
<p>Updated</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1571/discussion/#comment-1912052449'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by spradeep28** [comment]
<p>Updated</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1571/discussion/#comment-1912052418'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by spradeep28** [comment]
<p>Updated</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1571/discussion/#comment-1912052399'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by eaidova — APPROVED** [review]
